### ISSUE TITLE
Confusing output with Demo

### ISSUE THREAD

**Issue by Beep-Boop-Bopa** [taesungp/contrastive-unpaired-translation#71]
Hey, so I basically followed the given demo to do CUT with the following commands in order:
-> bash ./datasets/download_cut_dataset.sh grumpifycat
-> python train.py --dataroot ./datasets/grumpifycat --name grumpycat_CUT --CUT_mode CUT
-> python test.py --dataroot ./datasets/grumpifycat --name grumpycat_CUT --CUT_mode CUT --phase train

I was hoping to get a results layout like below, because I thought that this was unpaired:
![image](https://user-images.githubusercontent.com/42900802/106072538-5deccd00-60d6-11eb-9d2a-19cf7fb25bc5.png)

But got a result layout like this:
![image](https://user-images.githubusercontent.com/42900802/106072628-84126d00-60d6-11eb-8bff-f288df849c8b.png)

This layout makes it look like a paired out, i.e. real_A is supposed to look as close as possible to real_B. Is it right? how do I make it give an unpaired output?

**Comment by JunlinHan** [comment]
No, it should be unpaired out.
Train it again or adjust the weights of losses.  ( I guess make lambda_GAN to 2 helps)

**Comment by Beep-Boop-Bopa** [comment]
@JunlinHan Thanks for the advice. I have looked at all of the training options. I don't understand what to change. Do you get an unpaired output?

**Comment by JunlinHan** [comment]
> 
> 
> @JunlinHan Thanks for the advice. I have looked at all of the training options. I don't understand what to change. Do you get an unpaired output?

Yes, I got some good results before ( The FID is around 93). This dataset is a small one. GANs usually requires a larger dataset for training. So it's common if you get unfavorable results. Look at the code in CUT_model and adjust the weights of losses can help.
